src/pages/ErrorPage/ErrorPage.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import PageLayout from '../../PageLayout/PageLayout';
-import styles from './ErrorPage.scss';
+import { DivError, ImageError } from './ErrorPage.Styles';
 import image from '../../../public/error.jpg';
 
 function ErrorPage() {
   return (
     <PageLayout>
-      <div className={styles.error}>
-        <img src={image} className={styles.error__img} alt="Error Pic" />
-      </div>
+      <DivError>
```

Comment:
You are free to come up with any name you like but use eventually Div ? That's weird. And actually does not express anything.